### DISCUSSION TITLE
Are Any Developers reading the issues ...  is everyone on vacation?

### DISCUSSION THREAD

**Discussion by golfingal72** [apollographql/router#1378] [Q&A] [answered]
I have seen no responses to any issues recently and wonder if everybody is on vacation?

**Comment by abernix** [comment, marked as answer]
There has been plenty of regular activity on the repository [every single day this week](https://github.com/apollographql/router/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc).  We have not responded to every issue and every discussion, nor are we always able to, and there's plenty of features and functionality for us to design and improve.

Please be patient and we will respond to what we can.  Please do not merely open a new discussion to check if we're on vacation simply because we haven't responded to what you'd like us to have responded to.  That isn't particularly productive to the project (in fact, it makes more stuff for us to answer) and it's not welcome.

Thanks for your patience.

**Reply by golfingal72** [reply]
thanks...
